recipes/conda-vendor/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "conda-vendor" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+source:
+  url: https://github.com/MetroStar/conda-vendor/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a9d99fb2bd5b815bbebc8628242debbcbff5f6a1948624e720a95ee91a9a4cc9
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - conda-lock
+  run:
+    - python
+    - conda-lock
```

Upstream also lists `pyyaml`:
```suggestion
    - conda-lock
    - pyyaml
```